### PR TITLE
build_bus_regions: move voronoi partition from `vresutils` to script

### PR DESCRIPTION
**Pull request by fneum**
Up for discussion: This would be a step towards eliminating all `vresutils` dependencies.

### CODE REVIEW

File: scripts/build_bus_regions.py [hunk 1]
```diff
@@ -61,6 +62,66 @@ def save_to_geojson(s, fn):
...
+                # for a MultiPolygon pick the part with the largest area
+                poly = max(poly.geoms, key=lambda pg: pg.area)
+            except:
+                pass
+            return poly
+        polygons = [demultipolygon(poly) for poly in polygons]
+
+    polygons_arr = np.empty((len(polygons),), 'object')
+    polygons_arr[:] = polygons
+    return polygons_arr
```
Comment (FabianHofmann) [inline scripts/build_bus_regions.py:122]:
```suggestion
    return np.array(polygons, dtype=object)
```

**Review by FabianHofmann — COMMENTED** [review]
Is the `no_multipolygons` tag important?

**Comment by fneum** [comment]
> Is the no_multipolygons tag important?

It's not used anywhere. Fine to remove in my opinion. We can add back in from `vresutils` should the need occur later.

File: scripts/build_bus_regions.py [hunk 2]
```diff
@@ -61,6 +62,55 @@ def save_to_geojson(s, fn):
     s.to_file(fn, driver='GeoJSON', schema=schema)
 
 
+def voronoi_partition_pts(points, outline, no_multipolygons=False):
```
Comment (FabianHofmann) [inline scripts/build_bus_regions.py:65]:
```suggestion
def voronoi_partition_pts(points, outline):
```

File: scripts/build_bus_regions.py [hunk 3]
```diff
@@ -61,6 +62,55 @@ def save_to_geojson(s, fn):
...
+    """
+    Compute the polygons of a voronoi partition of `points` within the
+    polygon `outline`. Taken from
+    https://github.com/FRESNA/vresutils/blob/master/vresutils/graph.py
+    Attributes
+    ----------
+    points : Nx2 - ndarray[dtype=float]
+    outline : Polygon
+    no_multipolygons : bool (default: False)
+        If true, replace each MultiPolygon by its largest component
```
Comment (FabianHofmann) [inline scripts/build_bus_regions.py:75]:
```suggestion
```

**Comment by fneum** [comment]
You happy?

**Review by FabianHofmann — APPROVED** [review]

**Pull request merged**